src/validation.cpp

```diff
@@ -1335,6 +1334,14 @@ int BlockManager::GetSpendHeight(const CCoinsViewCache& inputs)
     return pindexPrev->nHeight + 1;
 }
 
+const CBlockIndex* BlockManager::GetLastPruneBlock(const CBlockIndex* start_block) {
+    const CBlockIndex* block = start_block;
+    CHECK_NONFATAL(block);
```

Comment:
This function doesn't use any member of BlockManager, so could be stand-alone? Also, I think you can avoid this check by having the caller pass a `const CBlockIndex& start_block`.

Reply:
Thanks! I made the changes as suggested.